emii/pom.xml

```diff
@@ -7,56 +7,66 @@
     <packaging>war</packaging>
     <version>1.0.0</version>
 
-    <name>AODN Geoserver (war)</name>
-    <description>AODN/IMOS Geoserver v${project.version} with XSLT, no GZip and CSV/metadata plugin.</description>
+    <name>eMII Geoserver (war)</name>
+    <description>eMII Geoserver v${project.version} with XSLT, no GZip and CSV/metadata plugin.</description>
     <url>http://www.emii.org.au</url>
 
     <parent>
         <groupId>au.org.emii</groupId>
         <artifactId>geoserver</artifactId>
         <version>1.0.0</version>
-        <relativePath>../../pom.xml</relativePath>
     </parent>
 
     <dependencies>
+
+        <dependency>
+            <groupId>org.geoserver.web</groupId>
+            <artifactId>gs-web-app</artifactId>
```

Comment:
It's either for the overlay manager which is still used, or for the custom xslt code.